contents/job-create.py

```diff
@@ -25,16 +25,31 @@ def create_job_object(data):
 
     annotations = None
     if "annotations" in data:
-        annotations_array = data["annotations"].split(',')
```

Comment:
it could be better to keep this code to not break old jobs (maybe if the yaml parse failed, try with this)